Ports/qt6-qt5compat/package.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=qt6-qt5compat
+version=6.2.3
+workdir=qt5compat-everywhere-src-${version}
+useconfigure=true
+files="https://download.qt.io/official_releases/qt/$(cut -d. -f1,2 <<< ${version})/${version}/submodules/qt5compat-everywhere-src-${version}.tar.xz qt6-qt5compat-${version}.tar.xz 1cf89198cf2cf8a5c15336ccd69fa1f39b779feb64117d6bbf5509c21c123f53"
+auth_type="sha256"
+configopts=(
+    "-GNinja"
+    "-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
+    "-DCMAKE_CROSSCOMPILING=ON"
+    "-DQT_BUILD_TOOLS_WHEN_CROSSCOMPILING=ON"
+    "-DQT_HOST_PATH=/usr"
+    "-DQT_FEATURE_cxx20=ON"
+)
+depends=("qt6-qtbase" "libiconv")
```

Comment:
Please break this up to one per line (also single quotes)

Reply:
Same as above